output/stepper.eslint.txt

```diff
@@ -0,0 +1,30 @@
+> eslint --config ../../.eslintrc src
+
+The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.
+
+/home/travis/build/Talend/ui/packages/stepper/src/components/index.js
+  1:8  error  Using exported name 'Stepper' as identifier for default export  import/no-named-as-default
+
+/home/travis/build/Talend/ui/packages/stepper/src/components/Stepper.connect.js
+  1:1  error  'react-redux' should be listed in the project's dependencies. Run 'npm i -S react-redux' to add it  import/no-extraneous-dependencies
+  2:8  error  Using exported name 'Stepper' as identifier for default export                                      import/no-named-as-default
+
+/home/travis/build/Talend/ui/packages/stepper/src/components/StepperTransition.component.js
+  3:1  error  'react-transition-group' should be listed in the project's dependencies. Run 'npm i -S react-transition-group' to add it  import/no-extraneous-dependencies
+
+/home/travis/build/Talend/ui/packages/stepper/src/service/Stepper.reducer.js
+    3:1  error  'invariant' should be listed in the project's dependencies. Run 'npm i -S invariant' to add it  import/no-extraneous-dependencies
+  131:1  error  Prefer default export                                                                           import/prefer-default-export
+
+/home/travis/build/Talend/ui/packages/stepper/src/service/Stepper.reducer.test.js
+   81:21  error  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap
+  132:21  error  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap
+  146:17  error  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap
+  158:25  error  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap
+  216:25  error  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap
+  276:31  error  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap
+  277:25  error  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap
+  332:25  error  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap
+
+✖ 14 problems (14 errors, 0 warnings)
```

Comment:
maybe not good to start a new package with 14 errors?

Reply:
I will fix them indeed